pkg/linear-pools/contracts/tetu-linear-pool/TetuLinearPoolRebalancer.sol

```diff
@@ -54,9 +54,9 @@ contract TetuLinearPoolRebalancer is LinearPoolRebalancer, TetuShareValueHelper
 
     function _getRequiredTokensToWrap(uint256 wrappedAmount) internal view override returns (uint256) {
         // Since there's fixed point divisions and multiplications with rounding involved, this value might
-        // be off by one. We add one to ensure the returned value will always be enough to get `wrappedAmount`
-        // when unwrapping. This might result in some dust being left in the Rebalancer.
+        // be off by one. We use mulUp to add one and ensure the returned value will always be enough to get
```

Comment:
Just a minor suggestion here, since it won't _always_ add one but will always round up (which is what we actually need).
```suggestion
        // be off by one. We use mulUp to round up and ensure the returned value will always be enough to get
```